pkg/constants/constants.go

```diff
@@ -203,3 +204,6 @@ func BootTitleForRole(role, title string) (string, error) {
 		return "", errors.New("invalid role")
 	}
 }
+
+// DiskUUID is the static UUID for main disk identification
+var DiskUUID = uuid.NewV5(uuid.NamespaceURL, "KAIROS_DISK").String()
```

Comment:
Consumers of this will call it with something like `constants.DiskUUID` which is a bit misleading. This is not a real constant, it changes value on every invocation of the `kairos-agent` right? It could lead to bugs if someone uses this to find the main disk because it won't return the same value every time you call it.

Maybe make it a method somewhere with a clear name? (e.g. `GenerateDiskUUID`) ?

Reply:
nope, its always the same, it generates a predictable uuid based on the type and text. I know that usually uiuid is for random stuff lol, but this makes it so the disk uuid will always be the same, so in the future we can even identify the kairos install disk by this uuid.

Same with the partitions, they are now all set to a predictable uuid based on their label name so again, we could identify the kairos partitions via uuid directly :D

Reply:
gotcha. I wonder what these magic hashes are supposed to be:

```
	NamespaceURL  = Must(FromString("6ba7b811-9dad-11d1-80b4-00c04fd430c8"))
```

Reply:
https://datatracker.ietf.org/doc/html/rfc4122#appendix-C

The answer is always on the spec :P

For some reason all of them have the same UUID so its kind of moot which one to use lol